OpenRA.Mods.Common/Traits/Render/RenderSprites.cs

```diff
@@ -229,8 +229,8 @@ public static string NormalizeSequence(Animation anim, DamageState state, string
 			return sequence;
 		}
 
-		// Required by RenderSimple
-		protected int2 AutoSelectionSize(Actor self)
+		// Required by RenderSimple, WithSpriteBody and WithInfantryBody
+		public int2 AutoSelectionSize(Actor self)
```

Comment:
We'll eventually want to move / remove this, but that can wait until after we remove `RenderSimple`.